File: iterableapi/src/main/java/com/iterable/iterableapi/IterableInAppCloseAction.java
package com.iterable.iterableapi;

public enum IterableInAppCloseAction {
    BACK {
        @Override
        public String toString() {
            return "back";
        }
    },

    LINK {
        @Override
        public String toString() {
            return "link";
        }
    },

    OTHER {
        @Override
        public String toString() {
            return "other";
        }
    }
}

File: iterableapi/src/main/java/com/iterable/iterableapi/IterableInAppCloseAction.kt
package com.iterable.iterableapi

enum class IterableInAppCloseAction {
    BACK {
        override fun toString(): String {
            return "back"
        }
    },

    LINK {
        override fun toString(): String {
            return "link"
        }
    },

    OTHER {
        override fun toString(): String {
            return "other"
        }
    }
}
